### ISSUE TITLE
Models will apply pulid no matter it liked to pulid or not after first generation

### ISSUE THREAD

**Issue by sd2530615** [balazik/ComfyUI-PuLID-Flux#41]
i set 2 ksamplers with flux.dev.1.

one with pulid and one not ,and keeping seed/sampler/scheduler the same of course.

first generation it works well ,but bugs comes out with 2nd generation.

the ksampler with out pulid was "pollute" by pulid and applied pulid, thus all those 2 ksamplers outpu the same image.

it would fix even i remove all pulid things in workflow, only Restart the comfyui can fix it.

first image was the first generation with pulid, it goes normal and works well,

![first](https://github.com/user-attachments/assets/88fab683-76ee-4d89-aae0-829b437d0619)

2nd image that  node bugged that pulid will applied to all models even not linked with it
![2nd](https://github.com/user-attachments/assets/4949b259-ea62-4b11-abd6-534d6f1f7485)

**Comment by balazik** [comment]
Hi, when you look in the readme in issues you will see:

> When the ApplyPulidFlux node is disconnected after first run, the Flux model is still influenced by the node.

Yes, I know about the issue, it is caused by the way how is PuLID for Flux integrated into the model. I literally patch the loaded model, so even if you have a sampler without it will always apply the PuLID because the loaded Flux model is already patched in memory. I know this is wrong, but ComfyUI had no hooks that allowed me to override the original forward, so I made this hacky solution. I plan to release a new version, where I will try to use the intended model patcher to mitigate this but I don't know if it will work.

**Comment by sd2530615** [comment]
i see, thanks for you answering me and great work on this node.

**Comment by jaszczw** [comment]
My workaround is to set weight to 0, which disables the pullid and I have the same results as without it being applied - not perfect but works for my use case, maybe it will help someone.

**Comment by fly2outerspace** [comment]
> My workaround is to set weight to 0, which disables the pullid and I have the same results as without it being applied - not perfect but works for my use case, maybe it will help someone.我的解决方法是将权重设置为 0，这会禁用 pullid，并且我得到与未应用它相同的结果 - 并不完美，但适用于我的用例，也许它会对某人有所帮助。

Hey, with your inspiration, I found a way to temporarily solve the problem at the code
at pulidflux.py line 360, replace the original `return (model, )` with following:
```
weight = 0
cond = [torch.zeros(1, 32, 2048).to(device, dtype=dtype)]
```
could help the issue, @balazik
